pistol-round-won-convert-to-3-0.js

```diff
@@ -0,0 +1,65 @@
+/* Calculate whether kevlar was a waste of money for pistol rounds
+ *
+ * We need to track a few things for this
```

Comment:
Wanna remove this comment? Seems to be copypasta from the previous one :smile:

Reply:
True!